build.gradle

```diff
@@ -21,6 +22,7 @@ allprojects {
     group = 'org.opt4j'
    
     repositories {
+	jcenter() 
```

Comment:
Because of Mockito. It is really cool for mocking up stuff. And it enable actual test-driven development where you can test the high-level classes without touching or even having an implementation of the low-level modules

Reply:
Ok. Then leave it in. Is mockito not available on maven central?

Reply:
I'll check. To be honest I just used the gradle dependency that was in the Mockito tutorial